src/main/java/it/gov/pagopa/bizpmingestion/service/impl/PMEventToViewServiceImpl.java

```diff
@@ -202,7 +202,7 @@ private BizEventsViewUser buildUserView(PMEvent pmEvent, PMEventPaymentDetail pm
                 .transactionId("PM-" + pmEventPaymentDetail.getPkPaymentDetailId().toString() + "-" + ldt.getYear())
                 .taxCode(userDetail.getTaxCode())
                 .transactionDate(pmEvent.getCreationDate())
-                .hidden(isHidden)
+                .hidden(true)
```

Comment:
@alessio-acitelli @pasqualespica basta settare hidden a true in questo punto per l'ingestion silente?

Reply:
yes, right put comment near t modification to remember it